sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client_async.py

```diff
@@ -536,9 +540,12 @@ def _validate_certificate_issuer_properties(self, issuer, expected):
 
         # update certificate policy
         cert_policy = CertificatePolicyGenerated(
-            key_properties=KeyProperties(exportable=True, key_type="RSA", key_size=2048, reuse_key=False),
-            secret_properties=SecretProperties(content_type="application/x-pkcs12"),
             issuer_parameters=IssuerParameters(name="Self"),
+            key_properties=KeyProperties(exportable=True, key_type="RSA", key_size=3072, reuse_key=False),
```

Comment:
Just curious: why change the key size?

Reply:
There's actually no real reason, I was running into some bug a while ago and was experimenting with ways to resolve it and this is a remnant of that. going to get rid of that